lang/es.js

```diff
@@ -133,6 +133,9 @@ var language = {
     },
     "pl": {
       "title": "Polaco"
+    },
+    "hr": {
+      "title": "Croata"
```

Comment:
This should still be in English when we roundtrip as we need to rely on the translators to proofread rather than us doing it.

Reply:
Right, no problem, thanks for the heads up.